modules/adagioBidAdapter.js

```diff
@@ -145,58 +147,37 @@ export const GlobalExchange = (function() {
   };
 })();
 
-export function adagioScriptFromLocalStorageCb(ls) {
-  try {
-    if (!ls) {
-      logWarn(`${LOG_PREFIX} script not found.`);
-      return;
-    }
-
-    const hashRgx = /^(\/\/ hash: (.+)\n)(.+\n)$/;
-
-    if (!hashRgx.test(ls)) {
-      logWarn(`${LOG_PREFIX} no hash found.`);
-      storage.removeDataFromLocalStorage(ADAGIO_LOCALSTORAGE_KEY);
-    } else {
-      const r = ls.match(hashRgx);
-      const hash = r[2];
-      const content = r[3];
+/**
+ * @param {string|number} adagiojsVersion The external adagio.js script version to use
+ * @returns {string} https://script.4dex.io/a[/VERSION]/adagio.js
+ */
+export function computeAdagioScriptUrl(adagiojsVersion) {
+  let versionDir = `/${ADAGIOJS_VERSION_LOCK}`;
 
-      if (verify(content, hash, ADAGIO_PUBKEY, ADAGIO_PUBKEY_E)) {
-        logInfo(`${LOG_PREFIX} start script.`);
-        Function(ls)(); // eslint-disable-line no-new-func
-      } else {
-        logWarn(`${LOG_PREFIX} invalid script found.`);
-        storage.removeDataFromLocalStorage(ADAGIO_LOCALSTORAGE_KEY);
-      }
-    }
-  } catch (err) {
-    logError(LOG_PREFIX, err);
+  if (adagiojsVersion === ADAGIOJS_VERSION_LATEST) {
```

Comment:
Why do you have a condition for latest ?  
If you have `ADAGIOJS_VERSION_LATEST = latest` and `adagiojsVersion === ADAGIOJS_VERSION_LATEST`, then you can use as well ` versionDir = `/${adagiojsVersion}`;` ?  
What am I missing ?